lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -59,12 +60,21 @@ def parse_term_authority_response(id)
           results = extract_preds(filtered_graph, preds_for_term)
           consolidated_results = consolidate_term_results(results)
           json_results = convert_term_to_json(consolidated_results)
-          termhash = select_json_result_for_id(json_results, id)
+          termhash = select_json_result_for_id(json_results, normalize_id(id))
           predicates_hash = predicates_with_subject_uri(termhash[:uri])
           termhash['predicates'] = predicates_hash unless predicates_hash.length <= 0
           termhash
         end
 
+        def normalize_id(id)
+          return id if expects_uri?
+          authority_name.to_s.casecmp('loc').zero? ? id.delete(' ') : id
```

Comment:
This is special processing for LOC.  In the triple, the id is specified with a blank (e.g. `sh 123`).  But to request the entity by URI, the blank cannot exist (e.g. http://id.loc.gov/authorities/subjects/sh123`.  The URI is generated from the passed in id.  If the blank is not removed, an exception is thrown for a malformed URI.